java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/MacAddress.java

```diff
@@ -20,6 +21,9 @@
  */
 
 public class MacAddress implements OFValueType<MacAddress> {
+
+    private static final Pattern MAC_ADDRESS_PATTERN = Pattern.compile("^([0-9A-Fa-f]{2}[:-]){5}([0-9Aa-f-F]{2})$");
```

Comment:
This issue was brought up on stackoverflow and apparently the spec `does` this. I'm not saying its right and I have no issues enforcing some consistency.